README.md

```diff
@@ -1,26 +1,82 @@
 # Loop Exercises
 
 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
-2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num". 
+
+for (let num = 10; num > 0; num -= 1) {
+  console.log(num)
+}
+
+let num = 10
+while (num > 0) {
+  console.log(num --)
+}
+
+2. Write a `while` loop and a `for` loop that takes the variable "num", and iterates over all numbers from 0 to "num".
 For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
-3. Write a `while` loop and a `for` loop that takes the variable "num" and iterates over all numbers from 0 to "num". 
+
+let num = 10
+while (num < 20){
+(num ++)
+  if (num % 2 == 0)
+console.log (num + " is even")
+else
+console.log (num)
```

Comment:
A couple things here: You want to log `num + " is odd"` as well, and you want to start at 0, not 10.